projects/ion/src/lib/indicator/indicator.component.scss

```
@@ -0,0 +1,72 @@
+@import '../../styles/index.scss';
+
+.indicator-content {
+  max-width: 250px;
+  text-align: center;
+  border-radius: 15px;
+  border: 1px solid #6e7192;
```

```suggestion
  border: 1px solid #6e7192;
```

use a color of theme.